web/client/components/map/leaflet/Layer.jsx

```diff
@@ -45,6 +46,26 @@ const LeafletLayer = React.createClass({
         }
         this.updateLayer(newProps, this.props);
     },
+    shouldComponentUpdate(newProps) {
+        // the reduce returns true when a prop is changed
+        // optimizing when options are equal ignorning loading key
+        return !(["map", "type", "srs", "position", "zoomOffset", "onInvalid", "onClick", "options"].reduce( (prev, p) => {
```

Comment:
I noticed an increase of performances during debug, with many features in the map.